Sources/PubNub/PubNub.swift

```diff
@@ -629,6 +682,47 @@ extension PubNub {
     }
   }
 
+  /// Fetches historical messages of a channel with message actions
+  ///
+  /// Keep in mind that you will still receive a maximum of 100 messages
+  /// even if there are more messages that meet the timetoken values.
+  ///
+  /// Iterative calls to history adjusting the start timetoken is necessary to page
+  /// through the full set of results if more than 100 messages meet the timetoken values.
+  ///
+  /// - Parameters:
+  ///   - for: List of channels to fetch history messages from.
+  ///   - max: The max number of messages to retrieve.
+  ///   - start: Time token delimiting the start of time slice (exclusive) to pull messages from.
+  ///   - end: Time token delimiting the end of time slice (inclusive) to pull messages from.
+  ///   - metaInResponse: If `true` the meta properties of messages will be returned as well (if existing).
+  ///   - with: Additional network configuration to use on the request
+  ///   - respondOn: The queue the completion handler should be returned on
+  ///   - completion: The async result of the method call
+  public func fetchMessageHistoryWithMessageActions(
```

Comment:
See comment about `includeMessageActions` flag on the `fetchMessageHistory` function

Reply:
I can collapse this (on the publish) into the existing method.

Going forward is the preference to add additional method parameters to existing methods when performing similar functions regardless of the difference in server endpoint (history vs history-with-actions) or amount of calls required (publish and add action)?

Edit: Also wanted to add that we shouldn't try and collapse APIs that don't share a common response. History and history-with-actions isn't an example of this, but just more of a food-for-thought.